src/ng/animate.js

```diff
@@ -420,7 +420,13 @@ var $AnimateProvider = ['$provide', function($provide) {
        * @param {DOMElement} parent the parent element which will append the element as
        *   a child (so long as the after element is not present)
        * @param {DOMElement=} after the sibling element after which the element will be appended
-       * @param {object=} options an optional collection of options/styles that will be applied to the element
+       * @param {object=} options an optional collection of options/styles that will be applied to the element.
+       *   The object has the following properties:
```

Comment:
It can have the following properties, but it doesn't have to. The only requirement is that "from" must have a matching "to"